### PR TITLE
Fiberparams to health

### LINKED ISSUE #2082: Add fiber coin parameters to /health endpoint
- Proper name (e.g. "Skycoin")
- Ticker code / short name (e.g. "SKY")
- Coin hours name (e.g. "SKY Hours")

### PR DESCRIPTION
**Pull request by therealssj**
Fixes #2082 

Output:
screenshot 2019-02-10 at 12 23 25 pm

Changes:
- Add `display_name`,`coinhours_name`,`ticker` and `explorer_url` to `/health` endpoint.

Does this change need to mentioned in CHANGELOG.md?
yes

### CODE REVIEW

**Comment by therealssj** [comment]
`coin` works fine for the proper name imo.

**Comment by gz-c** [comment]
Skycoin is different from skycoin and used in different contexts, for example a uri prefix would be skycoin but a display name would be Skycoin

**Comment by Senyoret1** [comment]
Is this related to https://github.com/skycoin/skycoin/issues/1785 ? If it is, then the explorer URL and the Coin Market Cap ID would also be nedded. Additionally, some of the images displayed in the wallet would have to be added to the node and made accessible from a public URL.

**Comment by gz-c** [comment]
These should be part of `fiber.toml`.

@Senyoret1 yes, they are related to #1785. Can you enumerate additional parameters needed in that issue?

**Comment by therealssj** [comment]
@gz-c https://github.com/skycoin/skycoin/blob/develop/src/skycoin/parameters.go#L55
Then we should set `CoinName` through `fiber.toml` 
Though I can see why it was decided to not put it in `fiber.toml` before as these parameters don't really "define" the node per se

**Comment by gz-c** [comment]
CoinName is set on the command line, it can be left at that

**Comment by Senyoret1** [comment]
@gz-c I listed the necessary parameters in the issue (the comment is in https://github.com/skycoin/skycoin/issues/1785#issuecomment-461957270 ). However, there is a new complication due to https://github.com/skycoin/skycoin/issues/2136 : due to the deprecation of the Coin Market Cap public API, it is probably necessary to first define which method will be used to obtain the USD value.

Additionally, one thing I noticed is that in this PR the hours are named “SKY Hours”, but are not they supposed to be called ”Coin Hours”?

**Comment by gz-c** [comment]
I'm not sure what the canonical name for coin hours should be, but I think we use "Coins Hours" everywhere

File: CHANGELOG.md [hunk 1]
```diff
@@ -34,6 +34,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   Now all options of a cli command must only use `--` prefix instead of a mix of `--` and `-` prefixes.
   `-` prefix is only allowed when using shorthand notation.
 - Use an optimized `base58` library for faster address decoding and encoding.
+- Add `display_name, `coinhours_name` and `ticker` to `/health` endpoint
```
Comment (gz-c) [inline CHANGELOG.md]: Missing a backtick

File: cmd/skycoin/skycoin.go [hunk 2]
```diff
@@ -32,8 +32,14 @@ var (
 
 	logger = logging.MustGetLogger("main")
 
+	// DisplayName user-facing name
+	DisplayName = "Skycoin"
 	// CoinName name of coin
 	CoinName = "skycoin"
+	// CoinhoursName name of coinhours
+	CoinhoursName = "SKY Hours"
```
Comment (gz-c) [inline cmd/skycoin/skycoin.go]: Use "Coin Hours"

File: fiber.toml [hunk 3]
```diff
@@ -31,6 +31,9 @@ peer_list_url = "https://downloads.skycoin.net/blockchain/peers.txt"
 # create_block_max_transaction_size = 32 * 1024
 # create_block_max_decimals = 3
 # max_block_size = 32 * 1024
+# display_name = "Skycoin"
+# coinhours_name = "SKY Hours"
```
Comment (gz-c) [inline fiber.toml]: Use "Coin Hours"

File: src/skycoin/config.go [hunk 4]
```diff
@@ -668,6 +677,11 @@ func (c *NodeConfig) RegisterFlags() {
 	flag.BoolVar(&c.Arbitrating, "arbitrating", c.Arbitrating, "Run node in arbitrating mode")
 	flag.StringVar(&c.WalletCryptoType, "wallet-crypto-type", c.WalletCryptoType, "wallet crypto type. Can be sha256-xor or scrypt-chacha20poly1305")
 	flag.BoolVar(&c.Version, "version", false, "show node version")
+
+	flag.StringVar(&c.CoinName, "coin", "skycoin", "name of the coin")
```
Comment (gz-c) [inline src/skycoin/config.go]: Isn't coin name set a different way?

**Comment by Senyoret1** [comment]
I added some comments that may be relevant on https://github.com/skycoin/skycoin/issues/1785#issuecomment-463778489

**Pull request closed without merging**